examples/http-server/main_test.go

```diff
@@ -20,9 +22,14 @@ import (
 	"gofr.dev/pkg/gofr/testutil"
 )
 
-const host = "http://localhost:9000"
+var host string
 
 func TestIntegration_SimpleAPIServer(t *testing.T) {
+	host = fmt.Sprint("http://localhost:9000")
```

Comment:
I'm unsure I get the point

```suggestion
	host = "http://localhost:9000"
```